### PR TITLE
feat(next13-zustand-bulma): add counter component

### LINKED ISSUE #994: [Next13 starter kit with CSS modules] - Create a counter component
# Background
We want to launch each kit with an example of a component with state and good tests. We've chosen a counter for this reason.

# Acceptance
- [ ] Create a counter component
- [ ] The counter should have an increment, decrement, and reset button
- [ ] The counter should display its current value
- [ ] The counter should be controlled by the global state manager
- [ ] The counter should have a storybook story
- [ ] The counter should have a set of tests demonstrating how to verify all its features

# Mock
Screen Shot 2022-04-05 at 9 48 58 AM

### PR DESCRIPTION
**Pull request by BigAB**
## Type of change

Add a Counter Component to the `next13-zustand-bulma` starter kit

counter

This PR NOW relies on #1177 being merged first

## Summary of change
- added client counter component, test, and story
- put component in counter-example page

## Checklist

- [x] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [x] Changes for this new starter kit are being pushed to an integration branch instead of main
- [x] All dependencies are version locked
- [x] This fix resolves #994 
- [x] I have verified the fix works and introduces no further errors

### CODE REVIEW

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Tested it out locally and it looks good.
I did notice that for small tablet sizes the buttons are squished together. 
Maybe we can go to a column at that point

Screenshot 2023-03-14 at 12 57 25 PM

**Comment by BigAB** [comment]
> Tested it out locally and it looks good. I did notice that for small tablet sizes the buttons are squished together. Maybe we can go to a column at that point

@jdwilkin4  Fixed and force pushed again.

I had to fix a lot of things actually, including storybook, how to add global styles to storybook, and fix the tests to work with zustand.  But here it is:

Screenshot 2023-03-14 at 4 41 47 PM

Screenshot 2023-03-14 at 4 42 12 PM

With the addition of the tests, I actually  now need to wait for #1177 to be merged before I can merge this, I think.

File: starters/next13-zustand-bulma/.storybook/with-font-decorator.tsx
Comment (jdwilkin4) [inline starters/next13-zustand-bulma/.storybook/with-font-decorator.tsx:9]: A note for later:
I think we should add a sentence or two about this google font and decorator stuff because I believe this is new for Next 13

**Review by jdwilkin4 — APPROVED** [review]
tested it again locally, and it works for me 👍

**Pull request merged**
